src/js/logic.js

```diff
@@ -0,0 +1,43 @@
+import { render } from "./render";
+import { createChapterStructure } from "./chapterDOM";
+import { createChapterEnd } from "./chapterEnd";
+import { createStoryEnd } from "./storyEnd";
```

Comment:
Tested the game - import from storyEnd gives an error. In storyEnd there is `export default` syntax instead of `export {}` like in other files. You can either remove the curly braces from `import { createStoryEnd }` to make it work. Or you can change the `export default` syntax to `export {}` syntax in storyEnd. I prefer changing the export syntax for consistency reasons.

Reply:
I can't commit changes in this branch without commiting whole storyEnd file, so changes to import should be done with other PR's or a single commit to main.

Reply:
Alright